bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -65,8 +68,21 @@ chi_eff_n = convert.chi_eff(mass1_n, mass2_n, spin1z_n, spin2z_n)
 mchirp_n = convert.mchirp_from_mass1_mass2(mass1_n, mass2_n)
 eta_n = convert.eta_from_mass1_mass2(mass1_n, mass2_n)
 
+assert len(args.fit_param) == len(args.log_param)
+
 if args.make_template_kde:
-    sample_n = numpy.vstack((numpy.log(mchirp_n), eta_n, chi_eff_n)).T
+    param0 = eval(args.fit_param[0][:])
+    param1 = eval(args.fit_param[1][:])
+    param2 = eval(args.fit_param[2][:])
+    if args.log_param == ['False', 'False', 'False']:
+        sample_n = numpy.vstack((param0, param1, param2)).T
```

Comment:
So here, even if we do not know the number of parameters, we can still make a list or tuple of arrays of parameter values (via looping over the fit param names given on the command line) and feed that to vstack.